spec/lib/s3_storage_spec.rb

```diff
@@ -315,4 +315,29 @@
       end
     end
   end
+
+  describe "#healthy?" do
+    let(:head_bucket_params) { { bucket: bucket_name } }
+
+    it "returns true when s3 bucket is reachable" do
+      allow(s3_client).to receive(:head_bucket)
+        .with(head_bucket_params).and_return(Aws::EmptyStructure.new)
```

Comment:
Bizarrely it looks like it returns a `Seahorse::Client::Response` object which - when represented with inspect shows "#<struct Aws::EmptyStructure>"

Evidence

```
irb(main):023:0> resp = client.head_bucket({ bucket: "govuk-assets-integration" })
[Aws::S3::Client 200 0.035634 0 retries] head_bucket(bucket:"govuk-assets-integration")

=> #<struct Aws::EmptyStructure>
irb(main):024:0> resp.class
=> Seahorse::Client::Response
irb(main):025:0> resp.to_s
=> "{}"
irb(main):026:0> resp.inspect
=> "#<struct Aws::EmptyStructure>"
```

Reply:
Oh thank you!! I was extremely confused as to how the EmptyStructure managed to get a successful? method on it